src/components/BaseCommentForm.js

```diff
@@ -109,6 +151,36 @@ class BaseCommentForm extends React.Component {
             value={this.state.commentText}
             onChange={this.handleTextChange.bind(this)}
           />
+          <FormGroup className="comment-form__file">
+            <ControlLabel><FormattedMessage id="add_images"/></ControlLabel>
+            <div className="comment-form__select-button">
+              <label className="btn btn-default btn-sm" htmlFor="fileInput">
+                <FormattedMessage id="choose_images"/>
+              </label>
+              <input
+                 type="file"
+                 ref="images"
+                 id="fileInput"
+                 multiple
+                 style={{display: 'none', visibility: 'hidden'}}
+                 onChange={(e) => this.imageTooBig(e.target.files)}
+              />
+              <div className="comment-form__selected-images">
+                {this.state.imageTooBig
+                  ? <div className="comment-form__image-too-big">
+                    <FormattedMessage id="image_too_big"/>
+                  </div>
+                  : null}
+                {this.refs.images
+                  ? this.getSelectedImagesAsArray(this.refs.images.files).map(
+                      (image) =>
+                        <p style={image.size > 1000000 ? {color: 'red'} : null}>
+                          {image.name} {round((image.size / 1000000), 3) + 'MB'}
```

Comment:
Looks good! Though we probably should add a help text that tells the user how to select several images at once in the dialog (press Ctrl/Cmd/whatever), otherwise they will end up clicking the button again to add more images and wonder why the previously selected image was replaced (as I did at first).